blockchain/vm/internaltx_tracer.go

```diff
@@ -268,8 +268,6 @@ func (this *InternalTxTracer) step(log *tracerLog) error {
 		input := ""
 		if int(inOff.Int64()) >= log.memory.Len() {
```

Comment:
How about pushing down this code to `Slice` function?

Reply:
Applied. 1b28e83